services/appflowy-collaborate/src/rt_server.rs

```diff
@@ -68,15 +68,14 @@ where
       info!("CollaborationServer with actix-web runtime");
     }
 
-    let metrics_calculate = CollabRealtimeMetrics::default();
```

Comment:
This was the issue. I didn't notice that we already had an `Arc` reference to properly initialised `CollabRealtimeMetrics` and instead was creating new one, not connected to registry.